### PR TITLE
intro and HPC section updates

### PR DESCRIPTION
**Pull request by eriksf**
- updated MPI container table
- stampede2 -> frontera
- make code-blocks consistent (console)

### CODE REVIEW

File: docs/containers/03.handsondocker.rst [hunk 1]
```diff
@@ -151,34 +151,34 @@ Alternatively, you could also do this *interactively*
 
 **Install your tool in the image**
 
-.. code-block:: bash
+.. code-block:: console
 
    root@ded8d40f1a1e:/#
-   # install dependencies
+   install dependencies
```
Comment (wjallen) [inline docs/containers/03.handsondocker.rst:157]: does this look better on readthedocs without the '#'?

File: docs/singularity/02.singularity_batch.rst [hunk 2]
```diff
@@ -25,11 +25,11 @@ On a HPC system, your job submission script would look something like:
 .. code-block:: bash
 
   #!/bin/bash
-  
+
   #SBATCH -J myjob                      # Job name
   #SBATCH -o output.%j                  # Name of stdout output file (%j expands to jobId)
-  #SBATCH -p normal                     # Queue name
-  #SBATCH -N 1                          # Total number of nodes requested (68 cores/node)
+  #SBATCH -p small                      # Queue name
```
Comment (wjallen) [inline docs/singularity/02.singularity_batch.rst:31]: we need to check with Lauren if the 'small' queue is the one she is asking for the reservation in

**Review by wjallen — APPROVED** [review]

File: docs/containers/03.handsondocker.rst (hunk 1 above)
Comment (eriksf) [reply docs/containers/03.handsondocker.rst:157]: Yes, I was testing this locally with the RTD theme as I was updating. With the 'console' code-block, anything that's not a prompt gets greyed out which looks nice. Unfortunately, the '#' is a typical root prompt, so it gets colored like a prompt.

File: docs/singularity/02.singularity_batch.rst (hunk 2 above)
Comment (eriksf) [reply docs/singularity/02.singularity_batch.rst:31]: I was wondering that as well (or if we'd have a reservation). In terms of cutting & pasting from the docs after the class, I figured either the 'small' or 'flex' queue made the most sense... 'development' is frequently full and the 'normal' queue requires a minimum of 3 nodes.

**Pull request merged**